### PR TITLE
IMAGEKIT-862-assetOwnership

### PR DESCRIPTION
**Pull request by yashaswi-imagekit**

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -372,7 +372,10 @@ class ImageKitUppyPlugin extends Plugin {
             }
             formData.append("publicKey", this.opts.publicKey);
             formData.append("file", file.data);
-
+            if(this.opts.createdBy)
```
Comment (niveditn) [inline src/main.js]: Should check for valid presence and length of `createdBy.userId` if we are following this spec:
```
createdBy: { 
  userId: "user-id" 
}
```

Comment (yashaswi-imagekit) [reply src/main.js]: if the 
length is not correct then what needs to be done?

File: src/main.js [hunk 2]
```diff
@@ -372,6 +372,26 @@ class ImageKitUppyPlugin extends Plugin {
             }
             formData.append("publicKey", this.opts.publicKey);
             formData.append("file", file.data);
+            if (this.opts.createdBy) {
+              if (
+                !this.opts.createdBy.userId ||
+                !this.opts.createdBy.userId.trim().length > 0
+              ) {
+                this.uppy.log(
+                  `UserId passed is invalid ${this.opts.createdBy.toString()}`
```
Comment (niveditn) [inline src/main.js]: `Missing or invalid userId` would be better. 

Also `${this.opts.createdBy.toString()}` would just print `"[object Object]"` in most cases. Let's remove it.

File: src/main.js [hunk 3]
```diff
@@ -372,6 +372,26 @@ class ImageKitUppyPlugin extends Plugin {
...
+                );
+
+                this.uppy.emit(
+                  "upload-error",
+                  file,
+                  buildResponseError(
+                    `UserId passed is invalid ${this.opts.createdBy.toString()}`
```
Comment (niveditn) [inline src/main.js]: Same as above.

**Review by niveditn — CHANGES_REQUESTED** [review]
Let's make the following minor changes and bump the version to next alpha.

`1.0.6-alpha.0` should be fine. Please check `npm pack` info once and DM me the output.

**Review by niveditn — APPROVED** [review]

**Pull request merged**